lib/payment/response_classes/dispute.rb

```diff
@@ -1,19 +1,25 @@
 module ShorePayment
   # Representation of an {Evidence} object in the Payment Service.
   class Evidence < StripeHash
-    attr_accessor :product_description, :customer_name, :customer_email_address,
-                  :billing_address, :receipt, :customer_signature,
-                  :customer_communication, :uncategorized_file,
-                  :uncategorized_text, :service_date, :service_documentation,
-                  :shipping_address, :shipping_carrier, :shipping_date,
-                  :shipping_documentation, :shipping_tracking_number
+    SUPPORTED_ATTRIBUTES = %i(
+      product_description customer_name customer_email_address billing_address
+      receipt customer_signature customer_communication uncategorized_file
+      uncategorized_text service_date service_documentation shipping_address
+      shipping_carrier shipping_date shipping_documentation
+      shipping_tracking_number
+    ).freeze
+
+    attr_accessor(*SUPPORTED_ATTRIBUTES)
```

Comment:
Not a big thing, but I actually prefer the original way this was written. It's more direct and with directness comes clarity.